release_creation/bundle/requirements/v1.8.pre.requirements.txt

```diff
@@ -1,16 +1,15 @@
-dbt-core~=1.7.0,>=1.7.3 --no-binary dbt-postgres
-dbt-snowflake~=1.7.0
-dbt-bigquery~=1.7.0
-dbt-redshift~=1.7.0
-dbt-postgres~=1.7.0
-dbt-spark[PyHive,ODBC]~=1.7.1
-dbt-databricks~=1.7.2
-dbt-trino~=1.7.0
-dbt-fabric~=1.7.0
-dbt-synapse~=1.7.0
-dbt-rpc~=0.4.1
-google-api-core<2.16.0 # see https://github.com/dbt-labs/dbt-bigquery/issues/1081
-pandas<2.2.0,>=2.0.0
+dbt-adapters~=1.0.0b1
```

Comment:
`dbt-adapters` and `dbt-common` are only here to allow pre-releases. There must be a dependency elsewhere that does not allow for it, but this solves it for now. Basically, we have:
```python
"dbt-adapters<2.0"
```
where we should have:
```python
"dbt-adapters>=0.1.0a1,<2.0"
```
Once we find where this is and we fix it, these two lines can (and should) be removed.